runner/lib/filter/mod.rs

```diff
@@ -0,0 +1,27 @@
+use std::fs::{File};
+use std::path::Path;
+use std::env;
+use seccompiler::{compile_from_json, BpfProgram, TargetArch};
+
+const FILTERS_CONFIG_PATH: &str = "config/filters.json";
```

Comment:
is it possible to make the path configurable, can we use the environment variable?